httprequest_lego_provider/management/commands/revoke_domains.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Revoke domains module."""
+
+# pylint:disable=imported-auth-user
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from httprequest_lego_provider.forms import FQDN_PREFIX
+from httprequest_lego_provider.models import Domain, DomainUserPermission
+
+# pylint:disable=duplicate-code
```

Comment:
I'm curios, does django require all of these commands to be in a separate file? Or could they be consolidated, e.g., into a `user_management` module or something like that?

Reply:
You could have one file with lots of different options that you pass to it for each command, but that wouldn't be very user-friendly, and any external project would be less likely to have done things in this way.